src/EasyRSA.php

```diff
@@ -165,14 +194,13 @@ public static function verify($message, $signature, PublicKey $rsaPublicKey)
      */
     protected static function rsaEncrypt($plaintext, PublicKey $rsaPublicKey)
     {
-        static $rsa = null;
-        if (!$rsa) {
-            $rsa = new RSA();
-            $rsa->setEncryptionMode(RSA::ENCRYPTION_OAEP);
```

Comment:
ah you mean the fact that the encryption mode is not always the same .. could we just make this a parameter on the getter that would then set the encryption mode before returning the RSA?